modules/mongodb/mongodb.go

```diff
@@ -40,10 +44,44 @@ func RunContainer(ctx context.Context, opts ...testcontainers.ContainerCustomize
 		return nil, err
 	}
 
+	username := req.Env["MONGO_INITDB_ROOT_USERNAME"]
+	password := req.Env["MONGO_INITDB_ROOT_PASSWORD"]
+	if username != "" && password != "" {
+		return &MongoDBContainer{Container: container, username: username, password: password}, nil
+	}
```

Comment:
Do we want to add some validation if one of the pair is empty? PTAL to MariaDB module for that.

Reply:
Added validation to return an error if one of the pair is empty. This was the behavior of MongoDB though. It fails to run if one of the pair is missing. But I think it is better to be explicit about it.